state_machine/applications/system-check/tests/sun_sensor_test.py

```diff
@@ -25,7 +24,7 @@ def lux(sun_sensor_index):
         return cubesat._cubesat.sun_xp.lux
     else:
         print(sun_sensor_index, "is not a defined sun sensor.")
-        return
+        raise ValueError
```

Comment:
`raise ValueError(f"Sun sensor index {sun_sensor_index} is not defined")`

Reply:
You can put the error message in the error rather than printing it here.

Reply:
Thanks!